android/modules/media/src/java/ti/modules/titanium/media/TiCameraActivity.java

```diff
@@ -602,7 +799,7 @@ private void checkWhichCameraAsDefault(){
 		if (backCameraId == Integer.MIN_VALUE && frontCameraId != Integer.MIN_VALUE) {
 			TiCameraActivity.whichCamera = MediaModule.CAMERA_FRONT;
 		} else {
-			TiCameraActivity.whichCamera = MediaModule.CAMERA_REAR;
+			//TiCameraActivity.whichCamera = MediaModule.CAMERA_REAR;				
```

Comment:
Because that would always set it to the rear camera even if you want to start it with the front camera. It would override the whichCamera parameter in openCamera